documentation/docs/pages/index.md

```diff
@@ -16,23 +21,27 @@ See the navigation links on the left. Some reading tips, see:
 ## access-am3-configs Overview
 ACCESS-AM3 configurations are provided via branches in the [access-am3-configs](https://github.com/ACCESS-NRI/access-am3-configs) GitHub repository, this repository is currently private. The [access-am3-configs](https://github.com/ACCESS-NRI/access-am3-configs) repository contains several configurations using the following components:
 
-- [MOM5](https://github.com/ACCESS-NRI/mom5) ocean model
 - [UM](https://github.com/ACCESS-NRI/UM) atmosphere model
+- [CABLE](https://github.com/CABLE-LSM/CABLE) land model
+- [JULES](https://github.com/ACCESS-NRI/JULES) land model
 
-All the configurations use the [Payu](https://payu.readthedocs.io/en/latest/) and rose/cylc workflow management tools, and pre-built executables available on [NCI](https://nci.org.au/).
+All the configurations use the [_Rose/Cylc_ workflow management tools](https://docs.access-hive.org.au/models/run_a_model/rose_cylc/) and pre-built executables available on [NCI](https://nci.org.au/).
 
 ### Repository structure
 
 Each configuration in [github.com/ACCESS-NRI/access-am3-configs](https://github.com/ACCESS-NRI/access-am3-configs) repository is stored as a git branch. Most of the branches are named according to the following naming scheme:
 
+* {dev|release}-{nominal_resolution}
 
-#### Supported configurations
+where {nominal_resolution} is the spectral resolution following the conventions from the UK Met Office (UKMO).
 
+#### Supported configurations
 
+* N96e global coupled atmosphere-land.
 
 #### How to use this repository to run a model
 
-All configurations use [payu](https://github.com/payu-org/payu) and rose/cylc to run the model.
+All configurations use [_Rose/Cylc_](https://docs.access-hive.org.au/models/run_a_model/rose_cylc/) to run the model.
 
 This repository contains many related experimental configurations to make support
 and discovery easier. As a user it does not necessarily make sense to clone all the
```

Comment:
Edit: I would simplify and combine the following 3 sentences as follows:
"In most cases, users will not need to clone all the configurations at once, but rather a single experiment. Simply choose which experiment, then run:"